### PR TITLE
feat(telemetry): server-side usage statistics

### PR DESCRIPTION
**Pull request by kevinhu**
## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: metadata-service/war/src/main/webapp/WEB-INF/beans.xml [hunk 1]
```diff
@@ -1,7 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <beans xmlns="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xmlns:context="http://www.springframework.org/schema/context"
-       xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-3.0.xsd http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context-3.0.xsd">
+       xmlns:task="http://www.springframework.org/schema/task"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans
```
Comment (jjoyce0510) [inline metadata-service/war/src/main/webapp/WEB-INF/beans.xml:5]: Why these changes?

Comment (jjoyce0510) [reply metadata-service/war/src/main/webapp/WEB-INF/beans.xml:5]: To get the scheduled task lib?

Comment (jjoyce0510) [reply metadata-service/war/src/main/webapp/WEB-INF/beans.xml:5]: And do there need to be changed to build.gradle?

Comment (kevinhu) [reply metadata-service/war/src/main/webapp/WEB-INF/beans.xml:5]: Yep, these are to get scheduled task functionality as described here: https://howtodoinjava.com/spring-core/spring-scheduled-annotation/ 

AFAIK there aren't any required changes to build.gradle and the scheduled functionality is working. @dexter-mh-lee can you confirm?
